# 不同方向的速度
float32 vel_n
float32 vel_e
float32 vel_d

# 运行角度
float32 targetYaw

# 运行时间：频率
float32 fly_time
